tests/functional/pv/test_qos_limits.py

```
@@ -0,0 +1,532 @@
+        log.info(
+            f"Target Pod is active on worker node: {node_name}. Polling cgroup io.max for limits..."
+        )
+
+        find_cmd = (
+            f"oc debug node/{node_name} -n default -- chroot /host "
+            f'sh -c \'find /sys/fs/cgroup/kubepods.slice/ -path "*pod{pod_cgroup_uid}*" '
+            f'-name "io.max" -type f -exec echo "===FILE: {{}}===" \\; -exec cat {{}} \\;\''
+        )
+
+        def _check_cgroup_limits():
+            res = exec_cmd(find_cmd, shell=True, ignore_error=True)
```

Ruff (0.16.1)

[error] Function call with `shell=True` parameter identified, security issue

(S604)